src/app/record/layout.tsx

```
@@ -209,12 +246,12 @@ export default function Layout({ children }: { children: React.ReactNode }) {
           onRightClick={isTodayTask && canGoDeepLog ? handleTodayTaskNextClick : undefined}
           rightDisabled={isTodayTask && (!canGoDeepLog || isTodayTaskSubmitting)}
           rightLabelClassName={
-            isTodayTask && canGoDeepLog && !isTodayTaskSubmitting ? "text-gray-100" : undefined
+            isTodayTask && canGoDeepLog && !isTodayTaskSubmitting ? "text-sea-blue-500" : undefined
           }
         />
       )}
 
-      <main className="flex min-h-0 flex-1 flex-col overflow-y-auto px-5 [-webkit-overflow-scrolling:touch]">
+      <main className="flex min-h-0 flex-1 flex-col overflow-y-auto px-5 [-webkit-overflow-scrolling:touch] [scrollbar-width:none] [&::-webkit-scrollbar]:hidden">
```

ESLint

[error] Replace `flex-col·overflow-y-auto·px-5·[-webkit-overflow-scrolling:touch]·[scrollbar-width:none` with `[scrollbar-width:none]·flex-col·overflow-y-auto·px-5·[-webkit-overflow-scrolling:touch`

(prettier/prettier)